### PR TITLE
chore: remove methods, that are implmented by PackageVersionTrait

### PR DESCRIPTION
**Pull request by lukadschaak**

### CODE REVIEW

File: src/NeustaPimcorePresentationBundle.php [hunk 1]
```diff
@@ -35,9 +25,4 @@ public function getInstaller()
     {
         return new Installer();
     }
-
-    protected function getComposerPackageName(): string
-    {
-        return 'teamneusta/pimcore-presentation-bundle';
-    }
```
Comment (jdreesen) [inline src/NeustaPimcorePresentationBundle.php]: This is still necessary until we set Pimcore `10.5` as minimum version (currently it's still `10.0`).

Comment (lukadschaak) [reply src/NeustaPimcorePresentationBundle.php]: Args. You said that

File: src/NeustaPimcorePresentationBundle.php [hunk 2]
```diff
@@ -35,9 +25,4 @@ public function getInstaller()
     {
         return new Installer();
     }
```
Comment (lukadschaak) [inline src/NeustaPimcorePresentationBundle.php:27]:
```suggestion
    }

    protected function getComposerPackageName(): string
{
    return 'teamneusta/pimcore-presentation-bundle';
}
```

File: src/NeustaPimcorePresentationBundle.php (hunk 1 above)
Comment (jdreesen) [reply src/NeustaPimcorePresentationBundle.php]: I think we should just raise the minimum version. There's no point in supporting Pimcore < 10.5 anymore.

Comment (lukadschaak) [reply src/NeustaPimcorePresentationBundle.php]: I re-added the part. This can be a general discussion later

**Review by jdreesen — APPROVED** [review]

**Pull request merged**